ios/FronteggRN.swift

```diff
@@ -93,13 +116,22 @@ class FronteggRN: RCTEventEmitter {
 
     @objc
     func login(
-      _ resolve: RCTPromiseResolveBlock,
-      rejecter reject: RCTPromiseRejectBlock
+      _ loginHint: String?,
+      resolver: @escaping RCTPromiseResolveBlock, rejecter: RCTPromiseRejectBlock
     ) -> Void {
+        
         DispatchQueue.main.sync {
-            fronteggApp.auth.login()
+            let completion: FronteggAuth.CompletionHandler = { result in
+                switch(result) {
+                case .success(_): 
+                    resolver("Success")
+                case .failure(let error):
+                    resolver("Failed: \(error.failureReason ?? "")")
+                        
+                }
+            }
+            fronteggApp.auth.login(completion, loginHint:loginHint)
```

Comment:
Did you got `Success` after login? I waiting for promise with no success

Reply:
fixed in ios native sdk